skyportal/handlers/api/instrument.py

```diff
@@ -333,6 +408,29 @@ def delete(self, instrument_id):
                         has no filters (e.g., because it is a spectrograph),
                         leave blank or pass the empty list.
                       default: []
+                    field_data:
+                      type: dict
+                      items:
+                        type: array
+                      description: |
+                        List of ID, RA, and Dec for each field.
+                    field_region:
+                      type: str
+                      description: |
+                        Serialized version of a regions.Region describing
+                        the shape of the instrument field
```

Comment:
Maybe say something about how you should only input the region or the fov type+attributes?

Reply:
Yup added.